TD.SseParserTest/FileParserFixture.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.IO;
+using System.Text;
+using TwoDave.ServerSentEventsParser;
+using Xunit;
+
+namespace TD.SseParserTest
+{
+    public class FileParserFixture
+    {
+        [Fact]
+        public void BeginningFileParserTest()
+        {
+            var input = @".\test.txt";
+
+            using(FileStream fs = File.Create(input))
+            {
+                Byte[] info = new UTF8Encoding(true).GetBytes("data: foo\r\ndata: bar\r\n\r\nevent: next\r\n");
```

Comment:
`UTF8Encoding` is usually accessed by `Encoding.UTF8` static or something like that.

Reply:
Changed